drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -1915,22 +1837,16 @@ IOAM proof of transit option of IOAM POT Type 0:
         via RFC Required process as per <xref target="RFC8126"></xref>.</t>
       </section>
 
-      <section title="IOAM Trace Flags Registry">
+      <section anchor="Flags-Registry-Sec" title="IOAM Trace Flags Registry">
         <t>This registry defines code points for each bit in the 4 bit flags
         for Pre-allocated trace option and Incremental trace option defined in
```

Comment:
s/flags for Pre-allocated trace option and Incremental trace option/for the Pre-allocated trace option and for the Incremental trace option/